internal/updatequota/controller.go

```diff
@@ -31,6 +31,7 @@ type UpdateQuotaReconciler struct {
 // +kubebuilder:rbac:groups=dana.hns.io,resources=updatequota/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=user.openshift.io,resources=groups,verbs=get;list;watch
 // +kubebuilder:rbac:groups="",resources=users,verbs=impersonate
+// +kubebuilder:rbac:groups=core,resources=configmaps,verbs=get;list;watch;create;update;patch;delete
```

Comment:
All controllers share a manager with one RBAC configuration so this line only needs to be added once